pkg/machines/scripts/create_machine.sh

```diff
@@ -53,16 +53,16 @@ if [ -z "$GRAPHICS_PARAM" ]; then
 fi
 
 if [ "$SOURCE_TYPE" = "pxe" ]; then
-    INSTALL_METHOD="--pxe --network $SOURCE"
+    INSTALL_METHOD="--pxe --network '$SOURCE'"
 elif [ "$SOURCE_TYPE" = "os" ]; then
     INSTALL_METHOD="--install os=$OS"
 elif [ "$START_VM" = "true" ]; then
     if [ "$SOURCE_TYPE" = "disk_image" ]; then
         INSTALL_METHOD="--import"
     elif ( [ "${SOURCE#/}" != "$SOURCE" ] && [ -f "${SOURCE}" ] ) || ( [ "$SOURCE_TYPE" = "url" ] && [ "${SOURCE%.iso}" != "$SOURCE" ] ); then
-        INSTALL_METHOD="--cdrom $SOURCE"
+        INSTALL_METHOD="--cdrom '$SOURCE'"
```

Comment:
It seems that this is the line which cause test failures.  I guess that virt-install also put's it's own quotes around the url, and then we get the error: 
`Validating install media ''/var/lib/libvirt/novell.iso'' failed: Could not start storage pool: cannot open directory '/run/user/1000/'/var/lib/libvirt':`

Reply:
Oha, thanks @skobyda! How weird, so the `$SOURCE` string includes quotes? That's quite a trap, but sure, I'll revert this then. But the `$STORAGE_POOL*` stuff should probably stay, as that is being used to construct a new path -- which wouldn't work if it already contained quotes.

Reply:
I still don't get this -- about half of the usage here in this file assumes it's not quoted, like

    elif ( [ "${SOURCE#/}" != "$SOURCE" ] && [ -f "${SOURCE}" ] ) || ( [ "$SOURCE_TYPE" = "url" ] && [ "${SOURCE%.iso}" != "$SOURCE" ] ); then

If our UI already pre-quotes the fourth argument to install_machine.sh, then the above would never work. (Note that it's *not* virt-install that does this -- the parameters are passed from the UI through install-machine.sh *to* virt-install)